doc/Advanced-configuration.md

```diff
@@ -398,6 +398,68 @@ If they are not present - defaults are used (connection to `localhost:9042` with
  ]}.
 ```
 
+#### SSL connection setup
+
+In order to establish a secure connection to Cassandra you must make some changes in the MongooseIM and Cassandra configuration files.
+
+##### Create server keystore
+Follow [this](https://docs.datastax.com/en/cassandra/3.0/cassandra/configuration/secureSSLCertWithCA.html) guide if you need to create certificate files.
+
+##### Change the Cassandra configuration file
+Find `client_encryption_options` in `cassandra.yaml` and make these changes:
+```
+client_encryption_options:
+    enabled: true
+    keystore: /your_certificate_directory/server.keystore
+    keystore_password: your_password
+```
+Save the changes and restart Cassandra.
+
+##### Enable MongooseIM to connect with SSL
+A secure connection can be established with both self-signed and CA-signed certificates.
+
+###### Self-signed certificate
+
+Find `cassandra_servers` in `ejabberd.cfg` and add the following line:
+```
+{cassandra_servers, [{default, [{ssl, [{verify, verify_none}]}]}]}.
+```
+Save the changes and restart MongooseIM.
+
+###### CA-signed certificate
+
+Find `cassandra_servers` in `ejabberd.cfg` and add the following line:
+```
+{cassandra_servers, [{default, [{ssl, [{cacertfile,
+                                        "$CRTS/rootCA.pem"},
+                                        {verify, verify_peer}]}]}]}.
+```
+Where `$CRTS` is the path to the directory containing the certificate. Save the changes and restart MongooseIM.
```

Comment:
i must say that usage of environment variable inside the erlang config file can be confusing and misinterpreted. you can not use environment variables here!

better use some fake path, like "/path/to/rootCA.pem"